static/js/components/CompactPostDisplay.js

```diff
@@ -84,9 +84,16 @@ export class CompactPostDisplay extends React.Component<Props> {
             </div>
             <div className="row">
               <div className="authored-by">
-                <Link to={profileURL(post.author_id)}>
-                  <div className="author-name">{post.author_name}</div>
-                </Link>
+                <div>
+                  <Link to={profileURL(post.author_id)}>
+                    <span className="author-name">{post.author_name}</span>
+                  </Link>
+                  {post.author_headline ? (
+                    <span className="author-headline">
+                      &nbsp;&#8212;&nbsp;{post.author_headline}
+                    </span>
+                  ) : null}
+                </div>
```

Comment:
Bit of a nitpick: I think instead of an extra div, we should try to make the HTML more semantically sensible. What do you think of something like this?

```html
<div class="column1">
<div class="row title-row">
  <a href="/c/analog_learning/1d/link_post">
    <div class="post-title">
      <span class="post-title">link post</span>
      <span class="url-hostname">(example.com)</span>
    </div>
  </a>
</div>
<div class="authored-by">
    <div class="row">
    	<a href="..."><span class="author-name">...</span></a>
      <span class="author-headline">...</span>
    </div>
    <div class="row">
		  <div class="date">a month ago</div>
    </div>
</div>
</div>
```

Reply:
That works too if I add `white-space: nowrap` to the `author-name` css.  But there is a slight difference in formatting that becomes evident at narrow browser widths, not sure which is better:

As is:
<img width="374" alt="screen shot 2018-08-10 at 1 01 07 pm" src="https://user-images.githubusercontent.com/187676/43971051-b110af88-9c9d-11e8-8eae-a0ae248c6640.png">

With proposed change:
<img width="376" alt="screen shot 2018-08-10 at 1 00 31 pm" src="https://user-images.githubusercontent.com/187676/43971064-ba25a7ae-9c9d-11e8-83c9-f98a10b779c7.png">

Reply:
OK. I'd say just leave it as-is. Not important enough to pay any more attention to it